Shared/AppInfo.swift

```diff
@@ -85,6 +85,6 @@ extension AppInfo {
     
     /// Return the keychain access group.
     public static func ecosiaKeychainAccessGroupWithPrefix(_ prefix: String) -> String {
-        return "\(prefix)\(".")+\(baseBundleIdentifier)"
+        return "\(prefix).\(baseBundleIdentifier)"
```

Comment:
Awesome find detective! 🕵️ Wondering where that `+` came from 🤔

Reply:
My guess is some copying and pasting from a place that did not do string interpolation 🙈

Reply:
Same here! 🤚 and your guess @lucaschifino is 99% what happened when coding that.